survey-app/routes/views/assessment.js

```diff
@@ -51,62 +49,8 @@ exports = module.exports = (req, res) => {
             };
         })).then((modules) => {
             locals.modules = modules;
-            console.log(modules);
             view.render('assessment');
         });
     });
-
-    view.on('post', async () => {
-        console.log(req.body);
-        const moduleId = req.body.moduleId;
-        const prevAnswers = await getUserAnswers(moduleId, req.user.id);
-
-        for (const question in req.body) {
-            if (question == 'moduleId') continue;
-
-            const answer = req.body[question];
-
-            // Textareas can submit empty strings
-            //if (answer) {
-                let previous = prevAnswers.find(_ => _.questionId == question);
-
-                if (previous && previous.answer !== req.body[question]) {
-                    const query = Answer.findOne({_id: previous._id});
-
-                    // findOneAndUpdate doesn't change updatedAt, so we're doing it manually...
-                    const updates = {
-                        answer: answer, 
-                        updatedAt: Date.now()
-                    };
-                    
-                    Answer.findOneAndUpdate(query, updates, {new: true}, (err, docs) => {
-                        if (err) {
-                            console.log(err);
-                        } else {
-                            console.log('worked?');
-                            console.log(docs);
-                        }
-                    });
-                } else {
-                    const newAnswer = new Answer({
-                        userId: req.user._id,
-                        questionId: question,
-                        moduleId: req.body.moduleId,
-                        answer: answer
-                    });
-
-                    newAnswer.save((err) => {
-                        if (err) {
-                            console.log('failed');
-                            console.log(err);
-                        } else {
-                            console.log('worked..?');
-                        }
-                    });
-                }
-            //}
-        }
-
-        return res.redirect('back');
```

Comment:
So I know this is totally crap for if we're auto submitting, but I think part of why this was here was so that the user got some sense of something happening when they click the button. 

I think there was actually another option that kept you at the same spot in the page and didn't refresh, but at the time I thought refreshing and starting from the top seemed more like something happened.

Should there be something that lets users know that marking as complete worked? Like a check mark or something?

Reply:
:+1: I'll add something to make it more obvious the button has been clicked!